src/components/post-ctrls/RecentChats.tsx

```diff
@@ -0,0 +1,161 @@
+          convos.map(convo => {
+            const otherMember = convo.members.find(
+              member => member.did !== currentAccount?.did,
+            )
+
+            if (!otherMember || otherMember.handle === 'missing.invalid')
+              return null
+
+            return (
+              <Button
+                key={convo.id}
+                onPress={() => onSelectChat(convo.id)}
+                label={_(
+                  msg`Send post to ${sanitizeDisplayName(
+                    otherMember.displayName || otherMember.handle,
+                  )}`,
+                )}
+                style={[
+                  a.flex_col,
+                  {width: 72},
+                  a.gap_sm,
+                  a.justify_start,
+                  a.align_center,
+                ]}>
+                <UserAvatar
+                  avatar={otherMember.avatar}
+                  size={56}
+                  type={otherMember.associated?.labeler ? 'labeler' : 'user'}
+                  moderation={moderateProfile(otherMember, moderationOpts).ui(
+                    'avatar',
+                  )}
+                />
+                <Text
+                  style={[
+                    a.text_xs,
+                    t.atoms.text_contrast_medium,
+                    a.text_center,
+                    {minHeight: 28},
+                  ]}
+                  numberOfLines={2}>
+                  {sanitizeDisplayName(
+                    otherMember.displayName || otherMember.handle,
+                  )}
+                </Text>
+              </Button>
+            )
+          })
+        ) : (
+          <>
+            <ConvoSkeleton />
+            <ConvoSkeleton />
+            <ConvoSkeleton />
+            <ConvoSkeleton />
+            <ConvoSkeleton />
+          </>
+        )}
+      </ScrollView>
+      {convos && convos.length === 0 && <NoConvos />}
+    </View>
+  )
+}
+
+function ConvoSkeleton() {
+  const t = useTheme()
+  return (
+    <View
+      style={[
+        a.flex_col,
+        {width: 72, height: 92},
+        a.gap_sm,
+        a.justify_start,
+        a.align_center,
+      ]}>
+      <View
+        style={[
+          t.atoms.bg_contrast_50,
+          {width: 56, height: 56},
+          a.rounded_full,
+        ]}
+      />
+      <View
+        style={[t.atoms.bg_contrast_50, {width: 56, height: 14}, a.rounded_xs]}
+      />
+    </View>
+  )
+}
+
+function NoConvos() {
+  const t = useTheme()
+
+  return (
+    <View
+      style={[
+        a.absolute,
+        a.inset_0,
+        a.justify_center,
+        a.align_center,
+        a.px_2xl,
+      ]}>
+      <View
+        style={[a.absolute, a.inset_0, t.atoms.bg_contrast_25, {opacity: 0.5}]}
+      />
+      <Text
+        style={[
+          a.text_sm,
+          t.atoms.text_contrast_high,
+          a.text_center,
+          a.font_bold,
+        ]}>
+        Start a conversation, and it will appear here.
```

Comment:
```suggestion
        <Trans>Start a conversation, and it will appear here.</Trans>
```
translation tags missing here :)